Dockerfile

```diff
@@ -2,30 +2,33 @@ FROM ubuntu:bionic
 
 RUN apt-get update \
   && apt-get install --no-install-recommends --yes --force-yes  \
-    locales \
```

Comment:
could you revert these tab space changes?

Reply:
cool. modified